content/en/docs/guides/Getting Started/Bringup/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Bringup Mardan Robot"
+linktitle: "Bringup"
+date: 2023-07-21
+weight: 2
+collapsible: false
+---
+
+{{< alert title="Bringup" >}}
+The code will be extracted from the [MardanRobotRpi](https://github.com/roboticamed/MardanRobotRpi) GitHub repository. 
+{{< /alert >}}
+
+### 1. Install Docker Compose  into RPi 4
+``` shell
```

Comment:
Missing blank line between Heading and paragraph, for this section and all below.

Reply:
Solved